mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/repository/ZuordnungRepository.java

```diff
@@ -0,0 +1,11 @@
+package de.muenchen.mobidam.repository;
+
+import de.muenchen.mobidam.domain.Zuordnung;
+import java.util.UUID;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
```suggestion

```

Die Annotation muss entfernt werden, da wir sonst eine unkontrollierte API für die Zuordnungen von Sprinng automatisch erstellt haben (siehe Bild)

![Screenshot 2023-11-10 120914](https://github.com/it-at-m/mobidam-sst-management/assets/77114015/fc4d4c8c-e621-4614-8438-2778e2ea558b)